deployment/ansible/roles/model-my-watershed.spark-jobserver/tasks/main.yml

```diff
@@ -37,7 +37,7 @@
     - Restart Nginx
 
 - name: Pull Spark Job Server container image
-  command: /usr/bin/docker pull quay.io/azavea/spark-jobserver:latest
+  command: /usr/bin/docker pull quay.io/azavea/spark-jobserver:0.6.1
```

Comment:
Would this be better placed in `group_vars` with the other version descriptions?